### PR TITLE
Install wheel before installing pillow

### PR DESCRIPTION
**Pull request by unnonouno**
I added `--disable-jpeg` option to install pillow.

### CODE REVIEW

**Comment by unnonouno** [comment]
I found wheel files of pillow in pypi.

File: run_test.py [hunk 1]
```diff
@@ -27,7 +27,9 @@
             'cuda': 'cuda65',
             'cudnn': 'cudnn2',
             'requires': [
-                'setuptools', 'cython==0.24', 'numpy<1.12', 'h5py', 'pillow'],
+                'setuptools', 'cython==0.24', 'numpy<1.12', 'h5py',
+                'wheel', 'pillow',  # wheel is required to isntall pillow
```
Comment (gwtnb) [inline run_test.py]: > get-pip.py will also install setuptools [3] and wheel, if they're not already. setuptools is required to install source distributions. Both are required to be able to build a Wheel Cache (which improves installation speed), although neither are required to install pre-built wheels.

https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py

**Comment by gwtnb** [comment]
conflict

**Comment by gwtnb** [comment]
LGTM

**Pull request merged**